proposal/plain-css-if.md

```diff
@@ -0,0 +1,463 @@
+
+### Evaluating an `IfGroup`
+
+This procedure takes an `IfGroup` `group` and returns either a boolean
+representing a result that's known at build time or a string representing a
+plain-CSS condition.
+
+* If `group` is a `SassCondition`, return a boolean indicating whether or not
+  the result of evaluating its expression is truthy.
+
+* Otherwise, if `group` contains a single `IfExpression`, return the result of
+  [evaluating that expression as an `IfCondition`].
+
+  [evaluating that expression as an `IfCondition`]: #evaluating-an-ifcondition
+
+* Otherwise, return the result of evaluating any interpolation in `group`.
+
+## Semantics
+
+### `IfExpression`
+
+* Let `results` be an empty list.
+
+* For each `IfBranch` `branch`:
+
+  * Let `condition` be the result of [evaluating `branch`'s condition].
+
+  * If `condition` is true:
+
+    * Let `value` be the result of evaluating `branch`'s expression.
+
+    * If `results` is empty, return `value`.
+
+    * Otherwise, add `["else", value]` to `results` and stop processing
+      additional `IfBranch`es.
+
+  * Otherwise, if `condition` is false, ignore this branch.
+
+  * Otherwise, let `value` be the result of evaluating `branch`'s expression and
+    add `[condition, value]` to `results`.
+
+  [evaluating `branch`'s condition]: #evaluating-an-ifcondition
+
+* If `results` is empty, return null.
+
+* Otherwise, return a string representation of a [CSS `if()` function] with the
+  given conditions (treated as unquoted strings) and values.
+
+  [CSS `if()` function]: https://drafts.csswg.org/css-values-5/#if-notation
+
+  > As usual, implementations are free to choose a string representation as long
+  > as it matches the CSS semantics of the function.
+
+## Functions
+
+### `if()`
+
+Remove the [top-level `if()` function].
+
+[top-level `if()` function]: ../spec/functions.md#if
+
+## Deprecation Process
+
+Because this proposal removes the old syntax for `if()`, it will be rolled out
+in two phases to give users plenty of time to upgrade.
+
+### Phase 1
+
+In this phase, both the old and the new syntax for `if()` will coexist.
+Specifically, an `IfExpression` can be parsed as a classic `IfExpression` if
+possible, and will fall back to being parsed using the new syntax. This is
+guaranteed to be safe because the new `if()` will always either contain a
+top-level `;` which is never legal in the old syntax *or* a single top-level `:`
+that's not preceded by any top-level commas. Since the old syntax takes three
+comma-separated arguments, if it contains a top-level `:` it will either have a
+comma before it or a `...` after it, neither of which is valid in the new
+syntax.
+
+In Phase 1, we will also retain the `if()` function as defined at the top level
+(which is distinct from the expression we handle at parse-time). However,
+evaluating either the top-level function or the expression will produce a
+deprecation error named `if-function`.
```

Comment:
This means that `meta.call(meta.get-function('if))` is deprecated, right ?
This is good news to me, given that this usage never had the semantic of a true conditional (as it evaluates all arguments eagerly).